tests/python/common/test_convert.py

```diff
@@ -0,0 +1,100 @@
+import backend as F
+import dgl
+
+
+def get_nodes_by_ntype(nodes, ntype):
+    return dict((k, v) for k, v in nodes.items() if v["ntype"] == ntype)
+
+
+def get_edges_by_etype(edges, etype):
+    return [e for e in edges if e[2]["etype"] == etype]
```

Comment:
Leave a comment explaining `e = (.., ..., ...)` so that we can better understand the magic number 2 here.

Reply:
I've added a method since this was used in several places.